equations.py

```diff
@@ -0,0 +1,51 @@
+###
```

Comment:
I know it looks nice but you don't need all those hashtags. optimization is key!